services/apps/src/apps/routes/admin.clj

```diff
@@ -153,6 +156,58 @@
           on the fields included in the request."
           (ok (apps/admin-update-category current-user (assoc body :id category-id)))))
 
+(defroutes* admin-ontologies
+
+  (GET* "/" []
+        :query [params SecuredQueryParams]
+        :return ActiveOntologyDetailsList
+        :summary "List Ontology Details"
+        :description
+"Lists Ontology details saved in the metadata service.
+
+#### Delegates to metadata service
+    GET /ontologies"
```

Comment:
Yeah, I'm not sure I like using `str` either, but I also don't like how repetitive these definitions are. I feel like my ideal one would be a more middleware-style invocation, such that it's something like `(delegated-endpoint-description ["metadata" :get "/ontologies"] "Lists Ontology details saved in the metadata service.")` (or something like that, details fuzzy). Not hugely important if we can't find a nice way to do it though; obviously this works, it's just a bit repetitive.